e2e/tests/general/general.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { generateWallet } from '../send/common';
+import { networkSwitch } from './network-switch';
+
+describe('General', () => {
+    /**
+     * Generate wallet
+     */
+    it('Generate wallet', async () => {
+        await generateWallet();
```

Comment:
if anything is needed before test execution you should do it in:
```
beforeAll(() => {...});
```
or if it's needed before each test:
```
beforeEach(() => {...})
```